user_docs/en/userGuide.md

```diff
@@ -42,15 +42,15 @@ Major highlights include:
 
 #### Minimum System Requirements {#MinimumSystemRequirements}
 
-* Operating Systems: all 32-bit and 64-bit editions of Windows 8.1, Windows 10, Windows 11, and all Server Operating Systems starting from Windows Server 2012 R2.
+* Operating Systems: 64-bit editions of Windows 10 and Windows 11.
+Windows Server 2016, 2019, 2022 and 2025.
   * both AMD64 and ARM64 variants of Windows are supported, including Copilot+ PCs.
-  * Note that 32-bit operating systems are no longer under active support.
-  * Note that Windows 8.1 and Windows Server versions older than 2022 are no longer under active support.
+  * Note that Windows 10 and Windows Server versions older than 2025 are no longer under active support.
```

Comment:
I still think the terminology here is ambiguous.
```suggestion
  * Note that Windows 10 and Windows Server versions older than 2025 are no longer under active support by NV Access.
```

Reply:
Fair, thanks for the suggestion